docs/devices/index.rst

```diff
@@ -35,7 +35,7 @@ The table below lists what devices and features are supported for each device.
 +------------------------------------+---------------+---------------+--------------+----------------+----------+----------+---------+----------+------------------+
 | P2WPKH Inputs                      | ✓             | ✓             | ✓            | ✓              | ✓        | ✓        | ✓       | ✓        | ✓                |
 +------------------------------------+---------------+---------------+--------------+----------------+----------+----------+---------+----------+------------------+
-| P2SH Multisig Inputs               | ✓             | ✓             | ✓            | ✓              | ✓        | ―        | ✓       | ✓        | ✓                |
+| P2SH Multisig Inputs               | ✓             | ✓             | ✓            | ✓              | ✓        | ✓        | ✓       | ✓        | ✓                |
```

Comment:
> P2SH Multisig is already supported, see

I think this line is about legacy-p2sh-multisig, i.e. non-segwit.
The code you link to is about "P2WSH_P2SH", i.e. witness v0 wrapped in p2sh, which is just below in the table, line 40.

It seems legacy-p2sh-multisig is not supported:
https://github.com/bitcoin-core/HWI/blob/a025af3ee3fdf0bac8dbdbc51b9e82ab946660b4/hwilib/devices/bitbox02.py#L511

Reply:
I'm afraid you are right! I'll close the PR, sorry.